types/folder/index.d.ts

```diff
@@ -22,7 +23,7 @@ export type SearchFolder = BaseFolder &
 	Pick<FolderFields, 'parent' | 'isLink'> &
 	SearchFolderFields;
 export type Folder = UserFolder | LinkFolder;
-
+export type RootFolder = Folder & { uuid: undefined };
```

Comment:
I have some doubt about the semantic of this type: if the _uuid_ should be always set to _undefined_ probably it should be removed from the type.
Maybe the type can be defined with an _Folder_ without the _uuid_ field (using the _Omit_ utility type)?

Reply:
the data is normalized by the folder worker and the value for uuid of a root is always 'undefined'. we could change the normalization in order to omit values that are not defined, but that should get it's own ticket as there are quite a few properties that follow the same logic and receive undefined as a value

Reply:
Ok, it makes sense. Probably the normalization should be changed but it will be something to be cover by another story